core/src/test/java/org/apache/calcite/rex/RexNormalizeTest.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.rex;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/** Test cases for {@link RexNormalize}. */
+class RexNormalizeTest extends RexProgramTestBase {
+
+  @Test void digestIsNormalized() {
+    assertNodeEquals(
+        and(or(vBool(1), vBool(0)), vBool(0)),
+        and(vBool(0), or(vBool(0), vBool(1))));
```

Comment:
Can you please add a test case like

```
    assertNodeEquals(
        and(or(vBool(1), vBool(0)), vBool(0)),
        and(or(vBool(0), vBool(1)), vBool(0))
    );
```

?

Does it pass?

Reply:
Added and passed.